specification/sbi_atee.adoc

```diff
@@ -1186,43 +1035,246 @@ The possible error codes returned in `sbiret.error` are shown below.
+-------
+
+Injects an external interrupt with the given `interrupt_id` into the specified vCPU. If the
+vCPU is presently bound to an IMSIC guest interrupt file, the interrupt is immediately
+injected by writing to the interrupt file. If it is not bound, the interrupt is recorded
+in the software and will be injected once the vCPU becomes bound. The specified interrupt ID
+must be valid and must have been allowed by the guest with `sbi_tee_guest_allow_external_interrupt()`.
+
+The possible error codes returned in `sbiret.error` are shown below.
+
+[#table_sbi_tee_interrupt_inject_tvm_cpu_errors]
+.TEE Interrupt Inject TVM CPU
+[cols="2,3", width=90%, align="center", options="header"]
+|===
+| Error code              | Description
+| SBI_SUCCESS             | The operation completed successfully.
+| SBI_ERR_INVALID_PARAM   | `tvm_guest_id` or `tvm_vcpu_id` or `interrupt_id` were invalid.
+| SBI_ERR_FAILED          | The operation failed for unknown reasons.
+|===
+
+[#sbi_tee_interrupt_rebind_tvm_cpu_aia_imsic_begin]
+=== Function: TEE Interrupt Rebind TVM CPU AIA IMSIC Begin (FID #8)
+[source, C]
+-------
+struct sbiret sbi_tee_interrupt_rebind_tvm_cpu_aia_imsic_begin(unsigned long tvm_guest_id,
+                                                               unsigned long tvm_vcpu_id,
+                                                               unsigned long imsic_mask);
+-------
+
+Begins the rebinding process for the specified vCPU to the current physical CPU and the specified
+confidential guest interrupt file. The host must complete a TLB invalidation sequence
+for the TVM before cloning the old interrupt file state using `sbi_tee_interrupt_rebind_tvm_cpu_aia_imsic_clone()`.
+Once cloned, the old file will be restored to the new guest interrupt file on 
+`sbi_tee_interrupt_rebind_tvm_cpu_aia_imsic_end()` invocation.
+
+The possible error codes returned in `sbiret.error` are shown below.
+
+[#table_sbi_tee_rebind_tvm_cpu_aia_imsic_begin_errors]
+.TEE Interrupt Rebind TVM CPU AIA IMSIC Begin
+[cols="2,3", width=90%, align="center", options="header"]
+|===
+| Error code              | Description
+| SBI_SUCCESS             | The operation was completed successfully.
+| SBI_ERR_INVALID_PARAM   | `tvm_guest_id` or `tvm_vcpu_id` or `imsic_mask` were invalid.
+| SBI_ERR_FAILED          | The operation failed for unknown reasons.
+|===
+
+[#sbi_tee_interrupt_rebind_tvm_cpu_aia_imsic_clone]
+=== Function: TEE Interrupt Rebind TVM CPU AIA IMSIC Clone (FID #8)
+[source, C]
+-------
+struct sbiret sbi_tee_interrupt_rebind_tvm_cpu_aia_imsic_clone(unsigned long tvm_guest_id,
+                                                               unsigned long tvm_vcpu_id);
+-------
+
+Clones the old guest interrupt file of the specified vCPU. The caller must make sure to invoke this from
+the old physical CPU. The guest interrupt file after this is free to be reclaimed or bound to another
```

Comment:
should be specific that the TSM clones the guest interrupt file.

Reply:
Fixed.